apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PathExpectationSamples.kt

```diff
@@ -29,12 +28,362 @@ class PathExpectationSamples {
+
+        fails {
+            expect(dir).toEndWith(Paths.get(INVALID_DIR))
+        }
+    }
+
+    @Test
+    fun notToEndWith() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir).notToEndWith(Paths.get(INVALID_DIR))
+
+        fails {
+            expect(dir).notToEndWith(Paths.get(TEST_DIR))
+        }
+    }
+
+    @Test
+    fun toExist() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir).toExist()
+
+        fails {
+            expect(Paths.get(INVALID_DIR)).toExist()
+        }
+    }
+
+    @Test
+    fun notToExist() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(Paths.get(INVALID_DIR)).notToExist()
+
+        fails {
+            expect(dir).notToExist()
+        }
+
+    }
+
+    @Test
+    fun toBeReadable() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir).toBeReadable()
+
+        fails {
+            expect(Paths.get(INVALID_DIR)).toBeReadable()
+        }
+    }
+
+    @Test
+    fun toBeWritable() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir).toBeWritable()
+
+        fails {
+            expect(Paths.get(INVALID_DIR)).toBeWritable()
+        }
+    }
+
+    @Test
+    fun toBeExecutable() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir).toBeExecutable()
+
+        fails {
+            expect(Paths.get(INVALID_DIR)).toBeExecutable()
+        }
+    }
+
+    @Test
+    fun toBeARegularFile() {
+        val file = tempDir.newFile(TEST_FILE)
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(file).toBeARegularFile()
+
+        fails {
+            expect(dir).toBeARegularFile()
+        }
+    }
+
+
+    @Test
+    fun toBeADirectory() {
+        val file = tempDir.newFile(TEST_FILE)
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir).toBeADirectory()
+
+        fails {
+            expect(file).toBeADirectory()
+        }
+    }
+
+    @Test
+    fun toBeAbsolute() {
+        val dir = tempDir.newDirectory(ABSOLUTE_DIR)
+
+        expect(dir).toBeAbsolute()
+
+        fails {
+            expect(Paths.get(RELATIVE_DIR)).toBeAbsolute()
+        }
+    }
+
+    @Test
+    fun toBeRelative() {
+        expect(Paths.get(RELATIVE_DIR)).toBeRelative()
+
+        fails {
+            expect(tempDir.newDirectory(ABSOLUTE_DIR)).toBeRelative()
+        }
+    }
+
+    @Test
+    fun toHaveTheDirectoryEntries() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+        val file = dir.newFile(TEST_FILE)
+
+        expect(dir).toHaveTheDirectoryEntries(TEST_FILE)
+
+        file.delete()
+
+        fails {
+            expect(dir).toHaveTheDirectoryEntries(TEST_FILE)
+        }
+
+    }
+
+    @Test
+    fun toHaveTheSameTextualContentAs() {
+        val writtenTextInFile = "test_test"
+
+        val notEmptyFilePath = tempDir.newFile("${TEST_FILE}_1")
+        notEmptyFilePath.toFile().writeText(writtenTextInFile)
```

Comment:
fine like this but you could also rewrite it to the following (using niok's writeText)
```suggestion
        notEmptyFilePath.writeText(writtenTextInFile)
```